File: abc/abc280/a.java
// https://atcoder.jp/contests/abc280/submissions/43238770

import java.util.Scanner;

class Main {
  public static void main(String[] args) {
    Scanner stdIn = new Scanner(System.in);

    int h = stdIn.nextInt();
    int w = stdIn.nextInt();

    String[] f = new String[h];
    for(int i = 0; i < h; i++) f[i] = stdIn.next();

    int ans = 0;
    for(int i = 0; i < h; i++) {
      for(int j = 0; j < w; j++) {
        if(f[i].charAt(j) == '#') {
          ans++;
        }
      }
    }

    System.out.println(ans);
  }
}

File: abc/abc280/a.cs
// https://atcoder.jp/contests/abc280/submissions/46673454

class A {
  static void Main() {
    string[] t = Console.ReadLine().Split(" ");
    int h = int.Parse(t[0]);
    int w = int.Parse(t[1]);

    int ans = 0;
    for(int i = 0; i < h; i++)
    {
      string s = Console.ReadLine();
      for(int j = 0; j < s.Length; j++)
      {
        if(s[j] == '#')
        {
          ans++;
        }
      }
    }

    Console.WriteLine(ans);
  }
}

